cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -224,6 +224,11 @@
     public static final String PROGRAM_CONTAINER_DIST_JARS = "program.container.dist.jars";
 
     public static final String APP_UPDATE_SCHEDULES = "app.deploy.update.schedules";
+
+    /**
+     * Comma separated TMS topics for scheduler
+     */
+    public static final String SCHEDULER_MESSAGING_TOPICS = "app.scheduler.messaging.topics";
```

Comment:
There can be more topics used for other types of notifications other than dataset partition notification.